### PR TITLE
Add support to set custom CA and define auth URLs to be skipped

### PR DESCRIPTION
**Pull request by phvalguima**

### CODE REVIEW

**Review by DnPlas — COMMENTED** [review]
Thanks @phvalguima, left a couple of comments. I think that in order to keep consistency with other podspec charms where SSL support is added, we should go with [this](https://github.com/canonical/minio-operator/pull/57) approach as it seems less prone to error when configurations do not exist. @ca-scribner thoughts?

File: src/charm.py [hunk 1]
```diff
@@ -61,33 +61,56 @@ def main(self, event):
...
+                        "CLIENT_ID": self.model.config["client-id"],
+                        "CLIENT_SECRET": secret_key,
+                        "DISABLE_USERINFO": True,
+                        "OIDC_PROVIDER": f"{public_url}/dex",
+                        "OIDC_SCOPES": oidc_scopes,
+                        "SERVER_PORT": port,
+                        "USERID_HEADER": "kubeflow-userid",
+                        "USERID_PREFIX": "",
+                        "SESSION_STORE_PATH": "bolt.db",
+                        "SKIP_AUTH_URLS": "/dex/" if len(skip_urls) == 0 else "/dex/," + skip_urls,
```
[the remark this answers is not in the document]
Comment (natalian98) [reply src/charm.py:83]: The list needs to be comma separated

**Comment by natalian98** [comment]
> Thanks @phvalguima, left a couple of comments. I think that in order to keep consistency with other podspec charms where SSL support is added, we should go with [this](https://github.com/canonical/minio-operator/pull/57) approach as it seems less prone to error when configurations do not exist. @ca-scribner thoughts?

Hi @dnplas thanks a lot for your review, I addressed your suggestions.
Regarding the approach used in minio, I think in that case we need to keep the config as `ca-bundle` as this is also required by oidc (see [optional settings](https://github.com/arrikto/oidc-authservice/blob/master/README.md)).

**Comment by DnPlas** [comment]
Thanks @natalian98 and @phvalguima, LGTM

**Review by DnPlas — APPROVED** [review]

**Pull request merged**